src/diff.ts

```diff
@@ -89,7 +92,8 @@ export function visualDomDiff(
     let diffOffset = 0
     let oldOffset = 0
     let newOffset = 0
-    ;({ done: diffDone, value: diffItem } = diffIterator.next())
+    diffItem = diffArray[diffIndex++]
+    diffDone = !diffItem
```

Comment:
I don't think `diffDone` is needed anymore - could `!diffItem` be used directly instead.

Reply:
Removed, thanks